src/toolboxes/paintbrush/Canvas.tsx

```diff
@@ -175,18 +171,23 @@ export class PaintbrushCanvasClass extends Component<Props, State> {
   };
 
   drawAllStrokes = (context = this.drawingCanvas.canvasContext): void => {
-    const { brushStrokes } = this.props.annotationsObject.getActiveAnnotation();
-
-    for (let i = 0; i < brushStrokes.length; i += 1) {
-      this.drawPoints(
-        brushStrokes[i].coordinates,
-        brushStrokes[i].brushColor,
-        brushStrokes[i].brushRadius,
-        brushStrokes[i].brushType,
-        false,
-        context
-      );
-    }
+    // Draw strokes on active layer whiles showing exiting paintbrush layers
+    this.props.annotationsObject
+      .getAllAnnotations()
+      .forEach((annotationsObject) => {
+        if (annotationsObject.toolbox === "paintbrush") {
+          for (let i = 0; i < annotationsObject.brushStrokes.length; i += 1) {
```

Comment:
If we are doing `.forEach` on 177, we should do the same here?

Reply:
Yup, it makes sense :)